pkgs/build-support/setup-hooks/generate-darwin-launcher.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
+fixupOutputHooks+=('generateDarwinLaunchers $prefix')
+
+# Get a param out of a desktop file. First parameter is the file and the second
+# is a pattern of the key who's value we should fetch.
+getDesktopParam() {
+    local file="$1";
+    local pattern="$2";
+
+    awk -F "=" "/${pattern}/ {print \$2}" "${file}"
+}
+
+# Various icons of different sizes are generated for desktop entries. Here we
+# use the filesize to select the largest icon (which will presumably have the
+# largest resolutioœn) for the launcher.
```

Comment:
Aha! Caught you trying to introduce a Unicode exploit, "œ" 👈 ; )

Reply:
Whops!  removed.